app/presenters/hyrax/manifest_components.rb

```diff
@@ -0,0 +1,78 @@
+module Hyrax
+  module ManifestComponents
+    delegate :rendering_ids, to: :solr_document
+
+    def manifest_url
+      manifest_helper.polymorphic_url([:manifest, self])
+    end
+
+    # IIIF rendering linking property for inclusion in the manifest
+    #
+    # @return [Array] array of rendering hashes
+    def sequence_rendering
+      renderings = []
+      if solr_document.rendering_ids.present?
+        solr_document.rendering_ids.each do |file_set_id|
+          renderings << build_rendering(file_set_id)
+        end
+      end
+      renderings.flatten
+    end
+
+    # IIIF metadata for inclusion in the manifest
+    #
+    # @return [Array] array of metadata hashes
+    def manifest_metadata
```

Comment:
What calls this method?  I only see tests calling it.

Reply:
The `iiif_manifest` gem calls it on the object it's given (in this case the presenter).